frontend/src/components/notifications/FollowedUserAddListNotification.vue

```diff
@@ -0,0 +1,95 @@
+<template>
+    <div class="notification-message">
+        <span class="user">
+            <span class="user__avatar-wrp">
+                <img class="user__avatar" :src="getUserAvatar">
+            </span>
+            <router-link
+                class="user__name"
+                :to="{ name: 'OtherUserPage', params: { id: user.id } }"
+            >
+                {{ getUserName }}
+            </router-link>
+        </span>
+        <span class="text">{{ $t('notifications.followed_user_add_list.add') }}</span>
+        <span class="list-link">
+            <router-link :to="`/lists/${notification['id']}`">
+                {{ this.notification.name }}
+            </router-link>
+        </span>
+        <span class="date" v-if="createdAt">
+            ({{ getDate }})
+        </span>
+    </div>
+</template>
+
+<script>
+import avatarStub from '@/assets/user-placeholder.png';
+
+export default {
+    name: 'FollowedUserAddListNotification',
+    props: {
+        notification: {
+            required: true,
+            type: Object
+        },
+        user: {
+            required: true,
+            type: Object
+        },
+        createdAt: {
+            default: null,
+            type: Object
+        }
+    },
+    data() {
+        return {
+            avatarStub: avatarStub
+        };
+    },
+    computed: {
+        getUserName() {
```

Comment:
getUserName(),  getUserAvatar(),  getDate() this duplicate methods could be a vue.js [mixin](https://vuejs.org/v2/guide/mixins.html)